pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dcicutils"
-version = "7.8.0"
+version = "7.8.0.1b0"
```

Comment:
Do you think it safe to update PyYML to ^6.0.1? I tried locally and tests pass, and I didn't see anything too scary in the release notes. It's just that everytime I build locally I have to manually update to this version due to Mac M1 issues with 5.4.1.

Reply:
Also, we can easily change to `">=5.1,<5.5,!=5.4.1"` if that would help you. Assuming it locks at all.